packages/remix-dev/cli/commands.ts

```diff
@@ -234,7 +234,11 @@ export async function watch(
   });
 }
 
-export async function dev(remixRoot: string, modeArg?: string) {
+export async function dev(
+  remixRoot: string,
+  modeArg?: string,
+  portArg?: number
+) {
```

Comment:
If we need to add one more arg to this in the future let's turn this into a single `options` argument.